src/bin/sol-fbp-generator/main.c

```diff
@@ -233,8 +232,7 @@ read_meta_type_file(const struct sol_flow_metatype_context *ctx,
     }
 
     contents = sol_file_reader_get_all(reader);
-    *buf = contents.data;
-    *size = contents.len;
+    *buf = SOL_BUFFER_INIT_CONST((char *)contents.data, contents.len);
```

Comment:
I'm not write sure here. I think the best option is append the data and not init the buffer.
@barbieri What do you think?